src/nvim/msgpack_rpc/server.c

```diff
@@ -52,33 +54,40 @@ static PMap(cstr_t) *servers = NULL;
 /// Initializes the module
 bool server_init(void)
```

Comment:
It looks like the one call-site for this function has an unchecked return value, although that might be out of scope for this PR.